common/kokkos-sampler/kp_sampler_skip.cpp

```diff
@@ -3,15 +3,19 @@
 #include <cstdlib>
 #include <cstring>
 #include <dlfcn.h>
+// #include <impl/Kokkos_Profiling_Interface.hpp>
 #include "../../profiling/all/kp_core.hpp"
 #include "kp_config.hpp"
 
+// using Kokkos::Tools::Experimental;
+// using mytpi_type = Kokkos::Tools::Experimental::ToolProgrammingInterface;
```

Comment:
Remove all commented lines you are not using.

Reply:
Thanks for that

Reply:
Fixed